admin/test/test_installer.py

```diff
@@ -0,0 +1,103 @@
+# Copyright ClusterHQ Inc.  See LICENSE file for details.
+"""
+Tests for ``admin.installer``.
+"""
+from textwrap import dedent
+
+from pyrsistent import PClass, field, pmap_field
+
+from flocker.testtools import (
+    TestCase
+)
+
+from ..installer._images import (
+    _packer_amis, _PackerOutputParser,
+)
+
+
+class ParserData(PClass):
+    input = field(type=unicode, mandatory=True)
+    output = pmap_field(key_type=unicode, value_type=unicode, optional=False)
+
+
+PACKER_OUTPUT_US_WEST_1 = ParserData(
```

Comment:
Would you mind adding a tiny blurb about where this text came from?

Looks like it's just output copied from a run of packer, but it'd be nice to make it explicit.